wpeverest/user-registration
# Tweak -  Save the user IP on the user meta.

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates the user IP in the user meta while registering and updating of profile from the edit profile.

### How to test the changes in this Pull Request:

1. Register a user and verify the user IP value on the user meta.
2. Login same user update the profile and verify the user IP value on the user meta.

### Types of changes:


### Other information:


### Changelog entry

> Tweak -  Save the user IP on the user meta..
